tests/test_quasi_topological_sort.py

```diff
@@ -0,0 +1,41 @@
+import networkx as nx
+from hypothesis import given, example
```

Comment:
Hey hey, what's this hypothesis thing? Is it a new dependency? We'll need to change angr-dev to account for it.

Reply:
setuptools also I believe has a way to mark test dependencies. We should probably have projects mark test dependencies rather than assume everyone always uses angr-dev.